x-pack/plugins/files/public/components/file_picker/i18n_texts.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const i18nTexts = {
+  title: i18n.translate('xpack.files.filePicker.title', {
+    defaultMessage: 'Select a file',
```

Comment:
The title is good if you can select 1 file.  If you can select multiple files, then the title should be "Select files",